### PR TITLE
*: add `START_TIME` and `UPTIME` columns to system table CLUSTER_INFO

### PR DESCRIPTION
**Pull request by lonng**
### What problem does this PR solve? 

This PR resolves issues https://github.com/pingcap/tidb/issues/14707

### What is changed and how it works?

1. Report the start timestamp of TiKV to PD (https://github.com/tikv/tikv/pull/6581).
2. Save TiKV start timestamp in PD and provide an HTTP API to retrieve the start timestamp of TiKV and PD (https://github.com/pingcap/pd/pull/2116).
3. Add two columns (`START_TIME` and `UPTIME`) to the system table `information_schema.cluster_info`

### Check List 

Tests 

 - Unit test
 - Integration test
 - Manual test (add detailed scripts or steps below)
```
mysql> select * from cluster_info;
+------+-----------------+-----------------+----------------------------------------------+------------------------------------------+---------------------------+-----------------+
| TYPE | INSTANCE        | STATUS_ADDRESS  | VERSION                                      | GIT_HASH                                 | START_TIME                | UPTIME          |
+------+-----------------+-----------------+----------------------------------------------+------------------------------------------+---------------------------+-----------------+
| tidb | 0.0.0.0:49920   | 0.0.0.0:12080   | 5.7.25-TiDB-v4.0.0-beta-100-g75bfcb6c5-dirty | 75bfcb6c5ddeac8f0d0a083929f306001ce69b5f | 2020-02-11T07:57:04+08:00 | 1m15.941958783s |
| tidb | 0.0.0.0:49900   | 0.0.0.0:10080   | 5.7.25-TiDB-v4.0.0-beta-100-g75bfcb6c5-dirty | 75bfcb6c5ddeac8f0d0a083929f306001ce69b5f | 2020-02-11T07:56:32+08:00 | 1m47.941966943s |
| pd   | 127.0.0.1:49901 | 127.0.0.1:49901 | 4.0.0-alpha                                  | d1cf01e084d40b7a07aa9a7c9890175a97815448 | 2020-02-11T07:56:27+08:00 | 1m52.94196873s  |
| tikv | 127.0.0.1:49903 | 127.0.0.1:49904 | 4.0.0-alpha                                  | 8af4016554b05de9334b1b821145fa92941b660d | 2020-02-11T07:56:37+08:00 | 1m42.941970325s |
+------+-----------------+-----------------+----------------------------------------------+------------------------------------------+---------------------------+-----------------+
4 rows in set (0.09 sec)
```

Release note

 - Add two columns (`START_TIME` and `UPTIME`) to the system table `information_schema.cluster_info`

### CODE REVIEW

File: domain/infosync/info.go [hunk 1]
```diff
@@ -79,8 +79,9 @@ type ServerInfo struct {
 
 // ServerVersionInfo is the server version and git_hash.
 type ServerVersionInfo struct {
```
Comment (wjhuang2016) [inline domain/infosync/info.go]: How about rename the struct name and update the comment?

Comment (lonng) [reply domain/infosync/info.go]: I have moved the `start_timestamp` to the struct `ServerInfo`.

**Pull request merged**